guidelines: add captions to verovio and listings for chapter 1–3

krHERO opened the pull request:
this PR adds missing captions to all listings for chapter 1–3

Refs https://github.com/music-encoding/music-encoding/issues/1217

krHERO committed:
add captions to chapter 1 and 2

Refs https://github.com/music-encoding/music-encoding/issues/1217

krHERO committed:
add captions to chapter 2

Refs https://github.com/music-encoding/music-encoding/issues/1217

krHERO committed:
add captions to chapter 3

Refs https://github.com/music-encoding/music-encoding/issues/1217

rettinghaus reviewed (changes requested):
Thanks for the great work. I noticed that some captions end with a full stop, others don't. It's a quite common practice to have none there (see <https://www.stylemanual.gov.au/content-types/images/alt-text-captions-and-titles-images>).

krHERO commented:
> https://www.stylemanual.gov.au/content-types/images/alt-text-captions-and-titles-images

thanks for pointing to this. I also was wondering what might be the "most correct" way how to do this and found some discussions stating, a full stop should be added when it is a full sentence and not to add one when it is just a word or some single words but no complete sentence. i tried to do this in this kind of way. 

since duden says "Auch Bildunterschriften werden nicht mit Punkt versehen, vor allem wenn es sich nicht um vollständige Sätze handelt. Bestehen Bildunterschriften allerdings aus mehreren Sätzen, wird im Allgemeinen der Punkt gesetzt." and i am absolutely not attached to this full stop, despite there is more than one sentence, i can easily update the commit by deleting all of them. 

would that be a suggestion that makes most of us happy?

krHERO committed:
update regarding full stop consistency

this commit deletes full stops. full stops are only provided, if the caption contains more than one sentence.

Refs https://github.com/music-encoding/music-encoding/pull/1635

krHERO commented:
i did an update regarding full stop. i am happy to get two reviews

rettinghaus commented on source/docs/01-introduction.xml:
```diff
@@ -588,7 +588,7 @@
                   <p>
                      <figure>
-                        <head/>
+                        <head>The <gi scheme="MEI">dynam</gi> element uses <att>startid</att> to reference the <att>xml:id</att> of the second quarter in the given measure</head>
```
Do we want/need the links to the elements in here?

krHERO replied:
i do not know, if they are wanted or needed. I just saw, that in another PR for issue #1217 there was a merged PR #1409 that also had one `<head>Example of an encoding without <gi scheme="MEI">measure</gi></head>`

krHERO replied:
since there are still a lot captions missing in the guidelines i would be very happy if we can decide if we want or need it and add this as an instruction or information to the issue #1217

musicEnfanthen commented:
@rettinghaus Is this good to go?

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -155,15 +155,15 @@
                   <p>
                      <figure>
-                        <head/>
+                        <head><gi scheme="MEI">title</gi> element</head>
```
```suggestion
                        <head>Example of a <gi scheme="MEI">title</gi> element</head>
```

krHERO replied:
thanks for the review and the your additions, which i can easily accept. just for explanation, why I wrote it like it was in some more cases where you everytime added "Example of". I referred to a [comment](https://github.com/music-encoding/music-encoding/pull/1409#issuecomment-1898869530) from perry from the PR of the mensural chapter "Or, simplest of all, just use "Augmentation". The reader can figure out that it's an example based on the visual context."

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -217,37 +217,37 @@
                   <p>
                      <figure>
-                        <head/>
+                        <head><gi scheme="MEI">respStmt</gi></head>
```
```suggestion
                        <head>Example of <gi scheme="MEI">respStmt</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -396,19 +396,19 @@
                         </figure>
                         <figure>
-                           <head/>
+                           <head>Detailed example of <gi scheme="MEI">pubStmt</gi></head>
```
```suggestion
                           <head>Detailed example of <gi scheme="MEI">pubStmt</gi> with <gi scheme="MEI">publisher</gi> and <gi scheme="MEI">distributor</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -573,7 +573,7 @@
                      <p>
                         <figure>
-                           <head/>
+                           <head><gi scheme="MEI">application</gi></head>
```
```suggestion
                           <head>Example of <gi scheme="MEI">application</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -598,7 +598,7 @@
                      <p>
                         <figure>
-                           <head/>
+                           <head><gi scheme="MEI">editorialDecl</gi></head>
```
```suggestion
                           <head>Example of <gi scheme="MEI">editorialDecl</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -615,7 +615,7 @@
                      <p>
                         <figure>
-                           <head/>
+                           <head><gi scheme="MEI">projectDesc</gi></head>
```
```suggestion
                           <head>Example of <gi scheme="MEI">projectDesc</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -637,18 +637,18 @@
                      <p>
                         <figure>
-                           <head/>
+                           <head><gi scheme="MEI">samplingDecl</gi></head>
```
```suggestion
                           <head>Example of <gi scheme="MEI">samplingDecl</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -698,14 +698,14 @@
                   <p>
                      <figure>
-                        <head/>
+                        <head><gi scheme="MEI">revisionDesc</gi></head>
```
```suggestion
                        <head>Example of <gi scheme="MEI">revisionDesc</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1064,7 +1064,7 @@
                   <p>
                      <figure>
-                        <head/>
+                        <head><gi scheme="MEI">langUsage</gi></head>
```
```suggestion
                        <head>Example of <gi scheme="MEI">langUsage</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1128,55 +1128,55 @@
                         </figure>
                         <figure>
-                           <head/>
+                           <head>Detailed <gi scheme="MEI">perfResList</gi></head>
```
```suggestion
                           <head><gi scheme="MEI">perfResList</gi> can group instrument or voice specifications (here woodwinds and strings section)</head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1203,35 +1203,35 @@
                   <p>
                      <figure>
-                        <head/>
+                        <head><gi scheme="MEI">contents</gi></head>
```
```suggestion
                        <head>Example of <gi scheme="MEI">contents</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1203,35 +1203,35 @@
                   <p>
                      <figure>
-                        <head/>
+                        <head><gi scheme="MEI">contents</gi> with <gi scheme="MEI">contentItem</gi></head>
```
```suggestion
                        <head>Example of <gi scheme="MEI">contents</gi> with <gi scheme="MEI">contentItem</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1203,35 +1203,35 @@
                   <p>
                      <figure>
-                        <head/>
+                        <head><gi scheme="MEI">contents</gi> with <gi scheme="MEI">contentItem</gi> and preceding optional <gi scheme="MEI">label</gi></head>
```
```suggestion
                        <head>Example of <gi scheme="MEI">contents</gi> with <gi scheme="MEI">contentItem</gi> and preceding optional <gi scheme="MEI">label</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1203,35 +1203,35 @@
                   <p>
                      <figure>
-                        <head/>
+                        <head><gi scheme="MEI">contents</gi> with <gi scheme="MEI">head</gi></head>
```
```suggestion
                        <head>Example of <gi scheme="MEI">contents</gi> with <gi scheme="MEI">head</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1261,21 +1261,21 @@
                   <p>
                      <figure>
-                        <head/>
+                        <head><gi scheme="MEI">termList</gi></head>
```
```suggestion
                        <head>Example of <gi scheme="MEI">termList</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1442,7 +1442,7 @@
                      <p>
                         <figure>
-                           <head/>
+                           <head>The <gi scheme="MEI">titlePage</gi> of Vaughan Williams' <hi rend="italic">On Wenlock Edge</hi></head>
```
```suggestion
                           <head>Example of <gi scheme="MEI">titlePage</gi> describing Vaughan Williams' <hi rend="italic">On Wenlock Edge</hi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1469,7 +1469,7 @@
                      <p>
                         <figure>
-                           <head/>
+                           <head><gi scheme="MEI">extent</gi></head>
```
```suggestion
                           <head>Example of <gi scheme="MEI">extent</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1479,7 +1479,7 @@
                      <p>
                         <figure>
-                           <head/>
+                           <head><gi scheme="MEI">dimensions</gi></head>
```
```suggestion
                           <head>Example of <gi scheme="MEI">dimensions</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1489,7 +1489,7 @@
                      <p>
                         <figure>
-                           <head/>
+                           <head><gi scheme="MEI">dimensions</gi> with <gi scheme="MEI">dim</gi></head>
```
```suggestion
                           <head>Example of <gi scheme="MEI">dimensions</gi> with <gi scheme="MEI">dim</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1500,15 +1500,15 @@
                      <p>
                         <figure>
-                           <head/>
+                           <head><gi scheme="MEI">watermark</gi></head>
```
```suggestion
                           <head>Example of <gi scheme="MEI">watermark</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1520,21 +1520,21 @@
                      <p>
                         <figure>
-                           <head/>
+                           <head><gi scheme="MEI">stamp</gi> with <gi scheme="MEI">corpName</gi> and <gi scheme="MEI">locus</gi></head>
```
```suggestion
                           <head>Example of <gi scheme="MEI">stamp</gi> with <gi scheme="MEI">corpName</gi> and <gi scheme="MEI">locus</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1520,21 +1520,21 @@
                      <p>
                         <figure>
-                           <head/>
+                           <head><gi scheme="MEI">stamp</gi></head>
```
```suggestion
                           <head>Example of <gi scheme="MEI">stamp</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1545,7 +1545,7 @@
                      <p>
                         <figure>
-                           <head/>
+                           <head><gi scheme="MEI">scoreFormat</gi></head>
```
```suggestion
                           <head>Example of <gi scheme="MEI">scoreFormat</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1616,7 +1616,7 @@
                         <p>
                            <figure>
-                              <head/>
+                              <head><gi scheme="MEI">foliaDesc</gi> with detailed placement information of recto and verso</head>
```
```suggestion
                              <head>Example of <gi scheme="MEI">foliaDesc</gi> with detailed placement information of recto and verso</head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1654,7 +1654,7 @@
                         <p>
                            <figure>
-                              <head/>
+                              <head><gi scheme="MEI">patch</gi></head>
```
```suggestion
                              <head>Example of <gi scheme="MEI">patch</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1714,7 +1714,7 @@
                         <p>
                            <figure>
-                              <head/>
+                              <head><gi scheme="MEI">plateNum</gi></head>
```
```suggestion
                              <head>Example of <gi scheme="MEI">plateNum</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1734,28 +1734,28 @@
                         <p>
                            <figure>
-                              <head/>
+                              <head><gi scheme="MEI">addDesc</gi></head>
```
```suggestion
                              <head>Example of <gi scheme="MEI">addDesc</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1734,28 +1734,28 @@
                         <p>
                            <figure>
-                              <head/>
+                              <head><gi scheme="MEI">addDesc</gi> with a little bit more structure</head>
```
```suggestion
                              <head>Example of <gi scheme="MEI">addDesc</gi> with a little bit more structure</head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1734,28 +1734,28 @@
                         <p>
                            <figure>
-                              <head/>
+                              <head><gi scheme="MEI">addDesc</gi> with <gi scheme="MEI">add</gi> and <att>hand</att> attribute</head>
```
```suggestion
                              <head>Example of <gi scheme="MEI">addDesc</gi> with <gi scheme="MEI">add</gi> and <att>hand</att> attribute</head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1844,7 +1844,7 @@
                   <p>
                      <figure>
-                        <head/>
+                        <head><gi scheme="MEI">repository</gi></head>
```
```suggestion
                        <head>Example of <gi scheme="MEI">repository</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1891,7 +1891,7 @@
                   <p>
                      <figure>
-                        <head/>
+                        <head><gi scheme="MEI">extMeta</gi></head>
```
```suggestion
                        <head>Example of <gi scheme="MEI">extMeta</gi></head>
```

musicEnfanthen commented on source/docs/03-metadata.xml:
```diff
@@ -1945,7 +1945,7 @@
                      <p>
                         <figure>
-                           <head/>
+                           <head><gi scheme="MEI">meiCorpus</gi></head>
```
```suggestion
                           <head>Example of <gi scheme="MEI">meiCorpus</gi></head>
```

krHERO committed:
Apply suggestions from code review

Co-authored-by: Stefan Münnich <stefan.muennich@unibas.ch>

krHERO commented:
thanks for your reviews, i accepted all suggestions and i am happy if you merge this PR :)

bwbohl reviewed (approved):
Many thank!

musicEnfanthen commented:
Thank you for the great work! :)
